src/utils_format_json.c

```diff
@@ -484,4 +495,215 @@ int format_json_value_list (char *buffer, /* {{{ */
         store_rates, (*ret_buffer_free) - 2));
 } /* }}} int format_json_value_list */
 
+#if HAVE_LIBYAJL
+static int json_add_string (yajl_gen g, char const *str) /* {{{ */
```

Comment:
The order *does* matter, which is exactly why I prefer this style: `const` always effects the keyword to its left: `char const *` is a variable pointer to a constant char, `char * const` is a constant pointer to a variable char.

When `const` is the first keyword, and only then, a special case kicks in and it effects the keyword to the right. I slightly prefer to avoid that special case because then constructs like `char const * const *ptr` become much easier to read (`const char * const *ptr` is just confusing).